src/boot.rs

```diff
@@ -126,6 +126,8 @@ pub async fn run_task<H: Hooks>(
 pub enum RunDbCommand {
     /// Apply pending migrations.
     Migrate,
+    /// Run one or more down migrations.
+    Down(Option<u32>),
```

Comment:
Following the CLI default value you can set the migration number will be not an option